### DISCUSSION TITLE
Changed Build: Mono Repo Setup & Yarn

### DISCUSSION THREAD

**Discussion by texttechne** [cpro-js/ui5-mobx#10] [Announcements]
The whole structure has been refactored to a mono repo.

I also switched from npm to yarn as package manager.

@wridgeu this is info is mainly intended for you. Also wanted to try out this discussion feature....

**Comment by wridgeu** [comment]
Cool feature, haven't used it myself as well. Good to see the switch to yarn and even yarn berry at that (if I saw that correctly). :) Are you actively using lerna in other projects for managing mono repos?

**Reply by texttechne** [reply]
yes, odata2ts 😄

it's probably overkill here though....
